3rdparty/libtengine/tengine.cmake

```diff
@@ -62,24 +62,17 @@ ENDIF()
 if(BUILD_TENGINE)
 	SET(HAVE_TENGINE 1)
 
-	# android system
-	if(ANDROID)
```

Comment:
is Android support removed?

Reply:
No . android still support. this just means  Android doesn't need to pass any compile special parameters.